### PR TITLE
Documented regex filter on senza

### PR DESCRIPTION
**Pull request by jmcs**

### CODE REVIEW

File: components/senza.rst [hunk 1]
```diff
@@ -47,10 +47,13 @@ Stacks can be listed using the ``list`` command:
...
-    $ senza list myapp.yaml --all # list stacks for myapp (also deleted ones)
-    $ senza list                  # list all active stacks
-    $ senza list --all            # list all stacks (including deleted ones)
+    $ senza list myapp.yaml         # list only active stacks for myapp
+    $ senza list myapp.yaml --all   # list stacks for myapp (also deleted ones)
+    $ senza list                    # list all active stacks
+    $ senza list --all              # list all stacks (including deleted ones)
+    $ senza list "suite-.*" 1       # list stacks starting with "suite" and with version "1"
+    $ senza list ".*" 42            # list all stacks  with version "42"
+    $ senza list mystack ".*-test"  # list all stacks for "mystack" with version ending in "-test"
```
Comment (hjacobs) [inline components/senza.rst]: versions cannot contain a dash, so the example does not make sense

File: components/senza.rst [hunk 2]
```diff
@@ -62,14 +65,19 @@ There are a few commands to get more detailed information about stacks:
...
 
-    $ senza inst            # all instances, no STACK_REF argument given
-    $ senza inst mystack    # list instances for all versions of "mystack"
-    $ senza inst mystack 1  # only list instances for "mystack" version "1"
+    $ senza inst                    # all instances, no STACK_REF argument given
+    $ senza inst mystack            # list instances for all versions of "mystack"
+    $ senza inst mystack 1          # only list instances for "mystack" version "1"
+    $ senza inst "suite-.*" 1       # list instances starting with "suite" and with version "1"
+    $ senza inst ".*" 42            # list all instances  with version "42"
+    $ senza inst mystack ".*-test"  # list all instances for "mystack" with version ending in "-test"
```
Comment (hjacobs) [inline components/senza.rst]: versions cannot contain a dash, so the example does not make sense

**Comment by jmcs** [comment]
@hjacobs can you check now?

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**